File: lab5v2.cpp
#include <iostream>
 
using namespace std;
 
int a, b, p, n, i;
int main(){

b= random()  ;
a= random()  ;

 n=b-a;
 
if (n >= 0)
{
     cout << "p: " ;
    for (i=0; i<=n; i++)
     {
    cout << a+i << ", ";
    }
}
else{
cout << "Error! a is bigger than b!";
}
return 0;
}

File: lab5v2.py
import random
a = random.randint(0,1000)
b = random.randint(0,1000)
i = 0

n=b-a

if (n >= 0):
    print("p: ")
    while i <= n:
         print( float(a+i), str(", "))
         i += 1
else:
    print("Error! a is bigger than b!")